example/user-guides/alerting/alertmanager-config-example.yaml

```diff
@@ -5,13 +5,57 @@ metadata:
   labels:
     alertmanagerConfig: example
 spec:
+  inhibitRules:
+  # critical > warning > info
+  #
+  # If an alert with severity 'critical'
+  # and an alert with severity 'warning' or 'info' occur at the same time,
+  # the alerts other than 'critical' will be ignored.
+  - sourceMatch:
+    - name: severity
+      value: critical
+    targetMatch:
+      - name: severity
+        value: warning
+      - name: severity
+        value: info
+    equal: [job]
+  - sourceMatch:
+    - name: severity
+      value: warning
+    targetMatch:
+      - name: severity
+        value: info
+    equal: [job]
+
   route:
-    groupBy: ['job']
+    groupBy: [job]
     groupWait: 30s
     groupInterval: 5m
     repeatInterval: 12h
-    receiver: 'webhook'
+    receiver: webhook
+    # Alerts are normally sent via webhook,
+    # but if an alert with severity 'critical' occurs,
+    # it will be sent through Telegram.
+    routes:
+    - match:
+      severity: critical
+      receiver: telegram
+      repeatInterval: 10m
```

Comment:
wrong indent
```suggestion
    receiver: telegram
    repeatInterval: 10m
```

Reply:
When applying this resource, there are no issues caused by indentation.
<img width="368" alt="Screenshot 2024-01-25 at 11 49 09 PM" src="https://github.com/prometheus-operator/prometheus-operator/assets/79129960/ea702a3b-aecb-4c1d-8a1d-dcd14d553838">
This screenshot is actually a config applied to the Alertmanager server after the example is applied.

Reply:
my bad.